libkbfs/prefetcher.go

```diff
@@ -83,14 +112,14 @@ func (p *blockPrefetcher) run() {
 }
 
 func (p *blockPrefetcher) request(priority int, kmd KeyMetadata, ptr BlockPointer, block Block) error {
-	// TODO: plumb through config, and then check the pointer and config data
-	// versions before prefetching anything. Use
-	// folderBlockOps.checkDataVersion as a template.
+	if err := checkDataVersion(p.config, path{}, ptr); err != nil {
+		return err
+	}
 	select {
 	case p.progressCh <- prefetchRequest{priority, kmd, ptr, block}:
 		return nil
 	case <-p.shutdownCh:
-		return io.EOF
+		return errors.Wrapf(io.EOF, "Skipping prefetch for block %v since the prefetcher is shutdown.", ptr.ID)
```

Comment:
No need to end with a period (I think one of the linters would complain about this actually, if we weren't using "errors" here).